ownCloud/Viewer/OCExtension Subclasses/OCDisplayExtension.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  OCDisplayExtension.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 03/09/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+
+class OCDisplayExtension: OCExtension {
+
+	init(identifier: OCExtensionIdentifier!, type: OCExtensionType!, location: OCExtensionLocation!, features: [String : Any]!, objectProvider: @escaping OCExtensionObjectProvider) {
```

Comment:
I was about to say that there's `+[OCExtension extensionWithIdentifier:type:location:features:objectProvider:]` already, but then noticed, that doesn't seem to get translated to Swift properly. 😬

I just added a pair of init-methods to the SDK (`feature/filemgmt`) now, so that this can be removed/replaced by them.